core/tauri/src/async_runtime.rs

```diff
@@ -123,7 +124,11 @@ mod tests {
   #[tokio::test]
   async fn handle_abort() {
     let handle = handle();
-    let join = handle.spawn(async { 5 });
+    let join = handle.spawn(async {
+      // Here we sleep 1 second to ensure this task to be uncompleted when abort() invoked.
+      tokio::time::sleep(tokio::time::Duration::from_secs(1)).await;
+      5
+    });
```

Comment:
It is irrelevant to this PR but I am gonna approve it anyway and wait for @lucasfernog to approve it too.

Reply:
Thank you for your review.

Reply:
yeah i just saw this test fail on windows, thanks for fixing it!